src/test/java/gov/di_ipv_fraud/step_definitions/FraudAPIStepDefs.java

```diff
@@ -0,0 +1,132 @@
+package gov.di_ipv_fraud.step_definitions;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import gov.di_ipv_fraud.pages.FraudPageObject;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.Base64;
+import java.util.Map;
+import java.util.logging.Logger;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class FraudAPIStepDefs {
+    private static String SESSION_REQUEST_BODY;
+    private static String SESSION_ID;
+    private final ObjectMapper objectMapper = new ObjectMapper();
+    private final int LindaDuffExperianRowNumber = 6;
+    private static final Logger LOGGER = Logger.getLogger(FraudPageObject.class.getName());
+
+    private String getPrivateAPIEndpoint(){
```

Comment:
Minor point but best practice for Java. We should keep private methods below all public methods